example/lib/main.dart

```diff
@@ -13,15 +13,20 @@ import 'dart:async';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
+
 import 'package:path_provider/path_provider.dart';
-import 'package:pspdfkit_flutter/pspdfkit.dart';
+
+import 'package:pspdfkit_flutter/pspdfkit_global.dart';
+import 'package:pspdfkit_flutter/pspdfkit_widget.dart';
 
 const String _documentPath = 'PDFs/PSPDFKit.pdf';
 const String _lockedDocumentPath = 'PDFs/protected.pdf';
 const String _imagePath = 'PDFs/PSPDFKit_Image_Example.jpg';
 const String _formPath = 'PDFs/Form_example.pdf';
 const String _instantDocumentJsonPath = 'PDFs/Instant/instant-document.json';
 const String _pspdfkitFlutterPluginTitle = 'PSPDFKit Flutter Plugin example app';
+const String _widgetExample = 'Embedded View Example';
+const String _widgetExampleSub = 'Opens a PDF Document as an embedded view.';
```

Comment:
"Embedded view" may not be a Flutter term, I think we should use the correct Flutter terminology.
 
```suggestion
const String _widgetExampleSub = 'Opens a PDF Document in the PSPDFKit widget.';
```